src/nicolai_munsterhjelm_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,32 @@
+def bubble_sort(data):
+    """This function sorts a tuple or list of numbers using the bubble sort algorithm.
+
+    Parameters
+    ----------
+    data (tuple or list)
+        The tuple/list to be sorted.
+
+    Returns
+    -------
+    new_list
+        Sorted list.
+    """
+
+    new_list = list(data)
```

Comment:
Try to think of a different name then `list`. As variable names generally shouldn't be named after their datatype.